InventoryInStorePickupSalesAdminUi/Block/Adminhtml/Order/View/ReadyForPickup.php

```diff
@@ -70,19 +70,19 @@ protected function _construct()
             return;
         }
 
-        $message = __(
-            'Are you sure you want to notify the customer that order is ready for pickup and create shipment?'
-        );
+        $message = $this->viewBlock->getOrder()->getShipmentsCollection()->getSize() > 0
```

Comment:
Can we avoid getting shipments via block, then order and then shipment? This seems too complicated and fragile. I would suggest using more appropriate APIs for this purpose